flake.nix

```diff
@@ -35,6 +35,11 @@
         description = "A default.nix and shell.nix for backward compatibility with Nix installations that don't support flakes";
       };
 
+      full = {
+        path = ./full;
+        description = "A template exercising the full flake schema".
```

Comment:
There really isn't such a thing as "the full flake schema", so maybe this should be something like "A template that shows all standard flake outputs".